### ISSUE TITLE
Aerosol-cloud coupling: constant Dv/Ka in ARG activation, and deposition INP is never read

### ISSUE THREAD

**Issue by duncanwp** [climate-analytics-lab/jax-gcm#679]
Found by a re-audit of the July 2026 ECHAM+RRTMGP review against `dev` @ `f6d1bcd`. Parent: #614.

Two defects on the aerosol→cloud pathway, found while checking that path against ESCOMP/CAM `cam6_4_196`.

**Context worth recording first:** the audit set out to test whether jcm pairs a *HAMMOZ* ARG implementation with *MAM4* modes — which would be a genuine incompatibility, since HAM uses a Köhler electrolyte `B` (only sulfate and sea salt ionic) where MAM4 supplies κ. **It does not.** jcm's ARG is a faithful port of **CAM's `ndrop.F90`** — `smc`, `maxsat`, `actfracs`, `f1`/`f2`, the volume-weighted κ-mixing rule and the `√(2/3·TKE)` single updraft all match CAM term-by-term, and every quantity feeding it (`number_vol` in #/m³ interstitial-only, `r_dry` from `calcsize`, κ, `sigma_g`) is a MAM4 quantity in the units CAM's ARG expects. The coupling is sound.

But `arg.py:4` documents it as mirroring HAMMOZ `mo_ham_activ::ham_activ_abdulrazzak_ghan`. **That attribution is wrong and actively hazardous**: it invites a future "fidelity fix" toward HAM, which would replace κ with `B` and `√(2/3·TKE)` with `0.7·√TKE` and break a working coupling. Correcting the docstring is part of this issue; the wider provenance sweep lives on #641.

## 1. The ARG growth coefficient uses constant `Dv` and `Ka`

`arg.py:141-144` hardcodes `_DV = 2.11e-5 m²/s` and `_KA = 0.024 W/m/K` — sea-level, room-temperature values. **Both** references make them T,p-dependent:

- CAM `ndrop.F90:1187-1188`: `diff0 = 0.211e-4*(p0/pres)*(tair/tmelt)**1.94`, `conduct0 = (5.69+0.017*(tair-tmelt))*4.186e2*1.e-5`
- HAM `mo_ham_activ.f90:244-258`: the same Pruppacher & Klett forms, plus a moist-air correction to `k`

Probed with a MAM4-like 3-mode population at w = 0.3 m/s:

| level | Dv_CAM / Dv_jcm | jcm N_act | with CAM's Dv, Ka |
|---|---|---|---|
| 900 hPa, 280 K | 1.18 | 107.0 cm⁻³ | 102.7 cm⁻³ (−4 %) |
| 700 hPa, 270 K | 1.42 | 109.3 cm⁻³ | 99.1 cm⁻³ (−9 %) |
| 500 hPa, 260 K | 1.84 | 113.8 cm⁻³ | 91.8 cm⁻³ (**−19 %**) |

Activation is biased **high, increasingly with altitude** — a systematic, sign-definite error sitting directly on the aerosol indirect effect, and cheap to fix.

Fold in the companion constant: `a_kelvin` uses σ = 0.0728 N/m (20 °C) evaluated at the actual T, vs CAM's 0.076 at a fixed 273 K (`ndrop.F90:45, 96`) and HAM's 0.075 at actual T. ~4 % low in A ⇒ ~6 % low in `sm`.

## 2. `ice_nuclei_deposition` is dead on the shipped default

`jcm/physics/aerosol/jam/ice_nucleation/` computes and publishes it, and `lohmann_2m/scheme.py:285` passes it as `newly_formed_ice` — but `assembly.py:445-451` reads it **only** under `nic_cirrus == 2`:

```python
if params.nic_cirrus == 1:
    icnc_candidate = 0.75/(pi*rhoice)*air_density*cloud_ice_in_cloud/max(ice_radius_mean**3, eps)
elif params.nic_cirrus == 2:
    icnc_candidate = jnp.minimum(newly_formed_ice, pressure*1.0e6)
```

with `nic_cirrus = 1` the default (`lohmann_2m_params.py:183`). So the entire **deposition-nucleation half of #494** — the Niemand/Ullrich `ns_dep` chain on insoluble dust and BC, its `deposition_scale` tunable, and its gradient — is computed every step and discarded.

This is adjacent to **#552** but distinct: #552 is "the `nic_cirrus = 2` source is hollow"; this is "a fully-populated producer feeds an input the default consumer branch never reads". Closing #552 would un-dead this input, so at minimum cross-reference it there.

**Checked and sound:** the immersion half of the contract is fine. Producer (`ice_term.py:99-100`, `max(inp_imm, 0)` in m⁻³ as a standing already-nucleated crystal number, with `niemand.py:50-51` capping by available particle number) and consumer (`scheme.py:335-337`, read as an m⁻³ standing ICNC target) agree on units and semantics, and the DeMott fallback is commensurate. The log-normal moment helpers in `population.py:102-121` are correct.

One design note for whoever fixes this: the "standing floor re-imposed every step, with no aerosol depletion and no rate" semantics differ from both references — CAM hands the microphysics a rate (`FRZIMM/FRZCNT/FRZDEP` in #/cm³/s) or a relaxation target (`naai`, relaxed via `nnuccd = (naai − ni/icldm)/mtime·icldm`), and ECHAM computes a rate internally. It is internally consistent for *number*, but it is exactly this framing that made it natural to omit the mass leg tracked in the 2M ledger issue. If that is fixed, add one comment explaining why re-imposing the same INP each step does not double-count mass (it does not, because `new_crystals = max(icnc_het − icnc_frz, 0)` is a difference) rather than leaving it to be re-derived.

## Refs

Review findings 2.24-a/b (re-scoped), and the new ARG constants finding. Related: #494, #552, #342, #341.
